Artifacts/windows-noop/Artifactfile.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "No-Op",
+    "publisher": "Microsoft",
+    "description": "Artifact internally used to work around Custom Script extension upgrade limitations on sysprepped images.",
```

Comment:
Confirm from PMs/Mark if we really want this artifact to have this description. Because I am not sure if we want to.